src/css/index.css

```diff
@@ -0,0 +1,103 @@
+/* TODO delete this probably
+:root{
+    --title-margin: 20;
+}*/
+
+* {
+    font-family: 'Times New Roman', Times, serif;
+}
+
+body {
+    margin: 60px;
+    margin-top: 0px;
+}
+
+button {
+    margin-bottom: 5px;
+    background-color: white;
+    border: 1px solid blue;
+    color: blue;
+    padding: 3px;
+}
+
+button:hover{
+    color: white;
+    background-color: blue;
+}
+
+.title-div {
+    text-align: center;
+    font-size: 40pt;
+    font-weight: bold;
+    margin-top: 25px;
+}
+
+.timeline-container {
+    border-style: solid;
```

Comment:
single line border

Reply:
Could you clarify this comment please?

Reply:
Oh, do you mean writing the shorthand border in one line?